commands/new_function.go

```diff
@@ -192,22 +199,22 @@ func printAvailableTemplates(availableTemplates []string) string {
 	return result
 }
 
-func duplicateFunctionName(functionName string, appendFile string) error {
-	fileBytes, readErr := ioutil.ReadFile(appendFile)
+func duplicateFunctionName(functionName string, yamlFile string) error {
```

Comment:
I know this was pre-existing but it could be better named.  It suggests that its going to produce a duplicate of something, when it is actually testing uniqueness.  It also returns an error when it finds a duplicate, which is counter-intuitive.